### PR TITLE
Clarify bias strength values and fix bias keys

### PR DESCRIPTION
**Pull request by jmbtutor**

### CODE REVIEW

**Comment by jmbtutor** [comment]
@willwarren @osman Corrections and clarifications to query-time biases documentation. Please review

File: src/main/java/com/groupbyinc/api/Query.java [hunk 1]
```diff
@@ -1254,7 +1254,7 @@ public Query addQueryUrlParams(String key, String value) {
      *  - `augmentBiases`: If true, the biases defined here will augment biasing profiles defined in Command Center.
      *  Otherwise, the biases will override the ones defined in command Center. By default, this is set to false.
      *  - `biases`: A list of biases, which either override or augment biasing profiles defined
-     *  in Command Center.
+     *  in Command Center. See {@link #addBias(String, String, Bias.Strength)} for more information.
```
Comment (codeallthethingz) [inline src/main/java/com/groupbyinc/api/Query.java]: We don't support JavaDoc links

**Comment by codeallthethingz** [comment]
@jmbtutor  minor change.

**Comment by jmbtutor** [comment]
@willwarren Change made.

**Pull request merged**